data_enrichment.py

```diff
@@ -0,0 +1,94 @@
+import json
+
+labels_file_path = "dags/dag_datalake_sirene/labels/"
+
+
+def load_file(file_name: str):
+    with open(f"{labels_file_path}{file_name}") as json_file:
+        file_decoded = json.load(json_file)
+    return file_decoded
+
+
+sections_NAF = load_file("sections_codes_naf.json")
+
+
+# Nom complet
+def create_nom_complet(nature_juridique_unite_legale=None, nom=None, nom_usage=None,
```

Comment:
when you do not have `sigle` or `formatted_sigle` you have something like `Pierre Dupont ()`. It's not a very big problem but not clean

Reply:
Yeah ! Nice catch, I added a `string.replace()`.